test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -233,7 +233,7 @@ def test_framework_version_cpu(image):
         else:
             if "neuron" in image:
                 assert tag_framework_version in output.stdout.strip()
-            if all(_string in image for _string in ["pytorch", "habana", "synapseai1.3.0"]):
+            if all(_string in image for _string in ["pytorch", "habana", "synapseai1.4.0"]):
```

Comment:
Will it make it backward incompatible for SynAI 1.3? If yes, lets modify the condition

Reply:
duplicated